src/url-history/parsers.ts

```diff
@@ -7,11 +7,12 @@ import qs from 'qs';
 type KeyableRootState = Omit< RootState, keyof EmptyObject >;
 
 // If you want any redux state to be tracked in the URL, add the top level key here.
```

Comment:
As an FYI: this is where all the meaningful code changes are! Everything else is just string type updates 👍